src/pages/home.tsx

```diff
@@ -8,7 +8,7 @@ export const HomePage: React.FunctionComponent = () => {
 
   return (
     <div className="bg-gray-100">
-      <DropZone setRawDocument={(doc) => setRawDocument(doc)} />
+      {rawDocument ? null : <DropZone setRawDocument={(doc) => setRawDocument(doc)} />}
```

Comment:
What if I want to drop another doc ? I need to reload the page? 

Keep they h-64 for the dropzone height and always display this component